### PR TITLE
Donors

### PR DESCRIPTION
**Pull request by adcpm**
Create donors page and add donor tag on user profile :

image

image

### CODE REVIEW

File: src/donors/Donors.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      <ul>
+        { Object.keys(donors).map(donor =>
+          <h3>
+            <Link to={`/@${donor}`}>
+              <Avatar username={donor} sm /> @{donor}
+            </Link>
+            { ' ' }
+            <Donor rank={donors[donor]} />
+            { ' ' }
+            <small></small>
```
Comment (nil1511) [inline src/donors/Donors.js]: for empty tags use it as `<small />` and why it this needed?

File: src/helpers/donors.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+export default {
+  'steemit': 'gold',
```
Comment (nil1511) [inline src/helpers/donors.js]: follow eslint rule. remove single quotes when not needed. this should be as steemit:gold

Comment (adcpm) [inline src/helpers/donors.js]: I used the single quote because it would not work for username with - or . character

**Comment by nil1511** [comment]
@adcpm not merge parallel branches.

File: src/helpers/donors.js (hunk 2 above)
Comment (nil1511) [inline src/helpers/donors.js]: use qoute only in name with - or . character.

**Pull request merged**